PR for TDR

quark2 opened the pull request:
- completion of new isolation codes
- reservation(?) of new isolations with R=0.4 (not available)

quark2 committed:
- branches will be fiiled by new iso values
- miniAOD list of TDR are added

quark2 committed:
Merge branch 'master' of https://github.com/jshlee/MuonPerformance
- for future use, new iso's with R=0.4 are added (not available yet)

Conflicts:
	MuonAnalyser/plugins/PatMuonAnalyser.cc

quark2 committed:
Merge branch 'master' of https://github.com/jshlee/MuonPerformance
- there is a minor conflict

Conflicts:
	MuonAnalyser/plugins/PatMuonAnalyser.cc

jshlee commented on MuonAnalyser/plugins/PatMuonAnalyser.cc:
```diff
@@ -297,7 +349,38 @@ void PatMuonAnalyser::fillBranches(TTree *tree, TLorentzVector &tlv, edm::RefToB
+    double relIso04 = (muon_puppiIso04NoLep_ChargedHadron+muon_puppiIso04NoLep_NeutralHadron+muon_puppiIso04NoLep_Photon)/muon->pt();
+    
+    b_muon_puppiNewIso_ChargedHadron = muon_puppiIsoNoLep_ChargedHadron;
+    b_muon_puppiNewIso_NeutralHadron = muon_puppiIsoNoLep_NeutralHadron;
```
b_muon_puppiNewIso_ChargedHadron = (*PUPPINoLeptonsIsolation_charged_hadrons)[muref];

quark2 committed:
adding variables becomes direct

quark2 committed:
a new option '-c' is added to create-batch (anything will not changed as before if you don't give this option)
 - if this option is given, the certification (from voms) will be sent to nodes via arguemnt 'x509userproxy'

jshlee commented on MuonAnalyser/test/patMuonAnalyser.py:
```diff
@@ -31,6 +31,11 @@
     pfIsolationChargedHadrons = cms.InputTag("muonIsolationMiniAOD","h+-DR030-ThresholdVeto000-ConeVeto000"),
     pfIsolationNeutralHadrons = cms.InputTag("muonIsolationMiniAOD","h0-DR030-ThresholdVeto000-ConeVeto001"),
     pfIsolationPhotons        = cms.InputTag("muonIsolationMiniAOD","gamma-DR030-ThresholdVeto000-ConeVeto001"),    
+    pfIsolationNeutralHadronsNoCone = cms.InputTag("muonIsolationMiniAOD","h0-DR030-ThresholdVeto000-ConeVeto001"),
```
this is the same as pfIsolationNeutralHadrons
